set.mm

```diff
@@ -801861,525 +801980,819 @@ Graph theory (extension)
-      anbi2d exbidv 3adant2 ad2antrr mpbird impd sylbid ) AKLZBKLZCDLZUDZABUEUF
-      ZBCUEUFZMAUGNZBUGNZEOZPZAUHNZUQZBUHNZUQZFOZPZWPUAOZWRNUIXDXBNWTNQUAWSRZMZ
-      FSZMZESZWOCUGNZGOZPZXACUHNZUQZHOZPZXKUBOZWTNUIXQXONXMNQUBXARZMZHSZMZGSZMA
-      CUEUFZWKWLXIWMYBWHWIWLXIUJWJABEFUAWRWTWNWOKKWNTZWOTZWRTZWTTZUKULWIWJWMYBU
-      JWHBCGHUBWTXMWOXJKDYEXJTZYGXMTZUKUMUNWKXIYBYCWKXHYBYCUOZEWKXHYJWKXHMZYAYC
-      GYKYAYCYKYAMZYCWNXJUCOZPZWSXNIOZPZYMJOZWRNZUIZYQYONZXMNZQZJWSRZMZISZMZUCS
-      ZYLUUFWNXJXKWPUPZPZYPUUHYRUIZUUAQZJWSRZMZISZMUCURUUHUUHURLYLXKWPGUSEUSUTV
-      AYLUUIUUNYAXLWQUUIYKXLXTVBWKWQXGVCWNWOXJXKWPVDVEYKXLXTUUNWKWQXGXLXTUUNUOZ
-      UOWKWQXLXGUUOWKWQXLXGUUOUOWKWQXLUDZXFUUOFUUPXFUUOUUPXFMZXSUUNHUUQXSUUNUUQ
-      XSMZUUMWSXNXOXBUPZPZUUJYQUUSNZXMNZQZJWSRZMIURUUSUUSURLUURXOXBHUSFUSUTVAUU
-      RUUTUVDXSXPXCUUTUUQXPXRVBUUPXCXEVCWSXAXNXOXBVDVEHGABCEFUAJUBDVFVIYOUUSQZY
-      PUUTUULUVDWSXNYOUUSVGUVEUUKUVCJWSUVEUUAUVBUUJUVEYTUVAXMYQYOUUSVJVHVKVLUNV
-      MVNVOVNVOVPVQVRVRVIYMUUHQZYNUUIUUEUUNWNXJYMUUHVGUVFUUDUUMIUVFUUCUULYPUVFU
-      UBUUKJWSUVFYSUUJUUAYMUUHYRVSVTVLWAWBUNVMWKYCUUGUJZXHYAWHWJUVGWIACUCIJWRXM
-      WNXJKDYDYHYFYIUKWCWDWEVNVOVNVOWFWG $.
-  $}
-
-  ${
-    strisomgrop.g $e |- G = <. V , E >. $.
-    strisomgrop.h $e |- H = { <. ( Base ` ndx ) , V >. ,
+      ( cuspgr wcel wa cv wex cpr cfv cgric wbr cgrim co wf1o wb wral c0 brgric
+      wne n0 bitri a1i isuspgrim exbidv bitrd ) ANOBNOPZABUAUBZCQZABUCUDZOZCRZF
+      GUSUEHQZIQZSDOVCUSTVDUSTSEOUFIFUGHFUGPZCRURVBUFUQURUTUHUJVBABUICUTUKULUMU
+      QVAVECHIEDUSABFGJKLMUNUOUP $.
+  $}
+
+  $( The "is isomorphic to" relation for graphs is a relation.  (Contributed by
+     AV, 11-Nov-2022.)  (Revised by AV, 5-May-2025.) $)
+  gricrel $p |- Rel ~=gr $=
+    ( cgric cvv cxp wss wrel cgrim ccnv cdif cima df-gric cnvimass grimfn fndmi
+    c1o cdm sseqtri eqsstri relxp relss mp2 ) ABBCZDUAEAEAFGBNHZIZUAJUCFOUAFUBK
+    UAFLMPQBBRAUAST $.
+
+  $( Graph isomorphism is reflexive for hypergraphs.  (Contributed by AV,
+     11-Nov-2022.)  (Revised by AV, 29-Apr-2025.) $)
+  gricref $p |- ( G e. UHGraph -> G ~=gr G ) $=
+    ( cuhgr wcel cid cvtx cfv cres cgrim co cgric wbr grimid brgrici syl ) ABCD
+    AEFGZAAHICAAJKALAAOMN $.
+
+  ${
+    $d R f $.  $d S f $.
+    $( Graph isomorphism implies the left side is a set.  (Contributed by AV,
+       29-Apr-2025.) $)
+    griclcl $p |- ( R ~=gr S -> R e. _V ) $=
+      ( vf cgric wbr cv cgrim co wex cvv c0 wne brgric n0 bitri grimdmrel ovrcl
+      wcel simpld exlimiv sylbi ) ABDEZCFZABGHZRZCIZAJRZUBUDKLUFABMCUDNOUEUGCUE
+      UGBJRABUCGPQSTUA $.
+
+    $( Graph isomorphism implies the right side is a set.  (Contributed by AV,
+       29-Apr-2025.) $)
+    gricrcl $p |- ( R ~=gr S -> S e. _V ) $=
+      ( vf cgric wbr cv cgrim co wex cvv c0 wne brgric n0 bitri grimdmrel ovrcl
+      wcel simprd exlimiv sylbi ) ABDEZCFZABGHZRZCIZBJRZUBUDKLUFABMCUDNOUEUGCUE
+      AJRUGABUCGPQSTUA $.
+  $}
```

Comment:
Probably shorter proofs using `brrelex1i` and `brrelex2i` ?

Reply:
You are right. But then these theorems become useless (two step proofs), therefore I will remove them (and they are not used anyway...).